ospd_openvas/wrapper.py

```diff
@@ -715,6 +716,23 @@ def get_detection_vt_as_xml_str(
 
         return tostring(_detection).decode('utf-8')
 
+    def sudo_check(self):
+        """ Checks that sudo is available and set the global var. """
+        _sudo_available = False
+        try:
+            result = subprocess.check_call(
```

Comment:
check_call doesn't return the returncode. If the returncode is != 0 it raises an exception https://docs.python.org/3/library/subprocess.html#subprocess.check_output So no need to check the result.